### ISSUE TITLE
Autobuilds: globbing tests failing

### ISSUE THREAD

**Issue by MarcelRaad** [curl/curl#11451]
### I did this

Look at the autobuilds at https://curl.se/dev/builds.html.

### I expected the following

Everything mostly green. Instead, the NetBSD 9.3 ARM and Solaris 10 SPARC runners both show failures in the following 5 tests:
0086 [HTTP, urlglob []-retrieval and -o #[num] usage]
0087 [urlglob with out of range -o #[num] usage]
1240 [glob [0-1] with stuff after range (7.33.0 regression)]
1283 [globbing range with same start and stop]
1328 [HTTP GET a globbed range with -f]

All of the failures are of the following pattern:
```
86: protocol FAILED:
--- log/check-expected 2023-07-16 04:09:16.958642306 +0200
+++ log/check-generated 2023-07-16 04:09:16.953213662 +0200
@@ -1,14 +1,14 @@
-GET /860001 HTTP/1.1[CR][LF]
+GET /0 HTTP/1.1[CR][LF]
```

### curl/libcurl version

master

### operating system

NetBSD 9.3 ARM and Solaris 10 SPARC (not i386!)

**Comment by bagder** [comment]
@MarcelRaad this looks fixed now, right?

**Comment by MarcelRaad** [comment]
Right, probably fixed by https://github.com/curl/curl/commit/20e3287aca994308cb040340d4d1865526cbf312, thanks!

**Comment by bagder** [comment]
Fixed via 20e3287aca994308cb040340d4d1865526cbf312
